web/assets/javascripts/metrics.js

```diff
@@ -0,0 +1,41 @@
+function init() {
+  const ctx = document.getElementById("metrics-chart");
+  const colors = JSON.parse(ctx.getAttribute("data-colors"));
+  const labels = JSON.parse(ctx.getAttribute("data-labels"));
+  const datasets = Object.entries(
+    JSON.parse(ctx.getAttribute("data-series"))
+  ).map(([k, v], i) => {
+    return {
+      label: k,
+      data: v,
+      borderColor: colors[i],
+      backgroundColor: colors[i],
+      borderWidth: 2,
+      pointRadius: 2,
+    };
+  });
+  const options = {
+    aspectRatio: 4,
+    scales: {
+      y: {
+        beginAtZero: true,
+      },
+    },
+    interaction: {
+      mode: "x",
```

Comment:
This configures the tooltip to show all points at the current spot on the x-axis, rather than just a single datapoint.

<img width="184" alt="CleanShot 2022-08-10 at 16 47 53@2x" src="https://user-images.githubusercontent.com/372/184017209-188cc192-d656-4901-afac-b004315750af.png">

versus...

<img width="206" alt="CleanShot 2022-08-10 at 16 48 25@2x" src="https://user-images.githubusercontent.com/372/184017233-d1de183e-760d-4f83-96d4-a965c0eb37ea.png">